### PR TITLE
FIX: Issues with editing images. [SDESK-5369]

### PR DESCRIPTION
**Pull request by GyanP**

### CODE REVIEW

File: superdesk/media/media_editor.py [hunk 1]
```diff
@@ -39,6 +39,10 @@ class MediaEditorResource(Resource):
             'type': 'dict',
             'required': True,
             'mapping': not_analyzed
+        },
+        'renditions': {
+            'type': 'dict',
+            'mapping': not_analyzed
```
Comment (petrjasek) [inline superdesk/media/media_editor.py:45]: I'm not sure about mapping here, it's not elastic resource anyhow so it shouldn't be used

Comment (GyanP) [reply superdesk/media/media_editor.py:45]: I'm also not sure, why it was done? Should I remove it for all?

Comment (petrjasek) [reply superdesk/media/media_editor.py:45]: I think so

Comment (GyanP) [reply superdesk/media/media_editor.py:45]: Ok.

**Pull request merged**